scripts/scripts.js

```diff
@@ -231,13 +231,15 @@ async function loadEager(doc) {
  */
 async function loadLazy(doc) {
   const main = doc.querySelector('main');
+
+  // eslint-disable-next-line no-unused-vars
+  loadHeader(doc.querySelector('header'));
```

Comment:
might affect CLS if we don't have a placeholder div that reserves the content area?

Reply:
we addressed it by leaving enough space for header to load

Reply:
I agree that the mega menu can be loaded lazily or even delayed! but the black bar of the header on desktop and the white bar on mobile are above the fold.. that part of the header should preferably be loaded eagerly instead of lazily. could be addressed as a follow up task.

cc @johan-t